RemoteHealthcare/BikeManager.cs

```diff
@@ -77,10 +74,8 @@ public async Task MakeConnectionAsync(string deviceName, int dataBlocks)
                     }
                 }
             }
-            return;
         }
 
-
         public bool CloseConnections()
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)